optaplanner-examples/src/main/java/org/optaplanner/examples/meetingscheduling/persistence/MeetingSchedulingGenerator.java

```diff
@@ -38,17 +38,16 @@
 import org.optaplanner.examples.meetingscheduling.domain.Room;
 import org.optaplanner.examples.meetingscheduling.domain.TimeGrain;
 import org.optaplanner.persistence.common.api.domain.solution.SolutionFileIO;
-import org.optaplanner.persistence.xstream.impl.domain.solution.XStreamSolutionFileIO;
 
 public class MeetingSchedulingGenerator extends LoggingMain {
 
     public static void main(String[] args) {
         MeetingSchedulingGenerator generator = new MeetingSchedulingGenerator();
         generator.writeMeetingSchedule(50, 5);
         generator.writeMeetingSchedule(100, 5);
-//        generator.writeMeetingSchedule(200, 5);
-//        generator.writeMeetingSchedule(400, 5);
-//        generator.writeMeetingSchedule(800, 5);
+        generator.writeMeetingSchedule(200, 5);
+        generator.writeMeetingSchedule(400, 5);
+        generator.writeMeetingSchedule(800, 5);
```

Comment:
The biggest file is 701Kb. That's acceptable, so +1 to make this change too

Reply:
Size is OK but I can't see why any user would want to try an example and wait 5 hours for a CH phase to finish:
```
$ 22:47:54.155 [EventQueue-0] INFO  Opened: data/meetingscheduling/unsolved/800meetings-2560timegrains-5rooms.xlsx
22:48:06.212 [l-2-thread-3] INFO  Solving started: time spent (34), best score (-1600init/-8528hard/0medium/0soft), environment mode (REPRODUCIBLE), random (JDK with seed 0).
22:48:13.247 [l-2-thread-3] DEBUG     CH step (0), time spent (7069), score (-1598init/-8523hard/0medium/-2soft), selected move count (12800), picked move ([Strategize data driven B2B {null -> R 1}, Strategize data driven B2B {null -> 0(Mon 08:00)}]).
22:48:18.718 [l-2-thread-3] DEBUG     CH step (1), time spent (12540), score (-1596init/-8519hard/0medium/-13soft), selected move count (12800), picked move ([Fast track sales driven e-business {null -> R 1}, Fast track sales driven e-business {null -> 4(Mon 09:00)}]).
22:48:22.968 [l-2-thread-3] DEBUG     CH step (2), time spent (16790), score (-1594init/-8516hard/0medium/-29soft), selected move count (12800), picked move ([Cross sell compelling virtualization {null -> R 1}, Cross sell compelling virtualization {null -> 13(Mon 11:15)}]).
22:49:49.336 [l-2-thread-3] DEBUG     CH step (3), time spent (103158), score (-1592init/-8486hard/0medium/-47soft), selected move count (12800), picked move ([Profitize reusable multitasking {null -> R 1}, Profitize reusable multitasking {null -> 18(Mon 13:30)}]).
22:50:12.346 [l-2-thread-3] DEBUG     CH step (4), time spent (126168), score (-1590init/-8474hard/0medium/-74soft), selected move count (12800), picked move ([Transform negotiated one stop shop {null -> R 1}, Transform negotiated one stop shop {null -> 20(Mon 14:00)}]).
22:50:38.424 [l-2-thread-3] DEBUG     CH step (5), time spent (152246), score (-1588init/-8460hard/0medium/-125soft), selected move count (12800), picked move ([Engage sustainable braindumps {null -> R 1}, Engage sustainable braindumps {null -> 36(Tue 08:00)}]).
22:51:02.980 [l-2-thread-3] DEBUG     CH step (6), time spent (176802), score (-1586init/-8446hard/0medium/-193soft), selected move count (12800), picked move ([Downsize laser-focused data mining {null -> R 1}, Downsize laser-focused data mining {null -> 53(Tue 13:15)}]).
22:51:06.102 [l-2-thread-3] DEBUG     CH step (7), time spent (179924), score (-1584init/-8444hard/0medium/-218soft), selected move count (12800), picked move ([Ramp up flexible policies {null -> R 2}, Ramp up flexible policies {null -> 4(Mon 09:00)}]).
22:51:22.184 [l-2-thread-3] DEBUG     CH step (8), time spent (196006), score (-1582init/-8434hard/0medium/-297soft), selected move count (12800), picked move ([On board real-time synergies {null -> R 1}, On board real-time synergies {null -> 72(Wed 08:00)}]).
22:51:54.268 [l-2-thread-3] DEBUG     CH step (9), time spent (228090), score (-1580init/-8418hard/0medium/-326soft), selected move count (12800), picked move ([Reinvigorate targeted user experience {null -> R 2}, Reinvigorate targeted user experience {null -> 6(Mon 09:30)}]).
22:52:17.077 [l-2-thread-3] DEBUG     CH step (10), time spent (250899), score (-1578init/-8406hard/0medium/-356soft), selected move count (12800), picked move ([Strategize data driven e-business {null -> R 1}, Strategize data driven e-business {null -> 29(Mon 16:15)}]).
```
10 CH steps in 4 minutes => 800 CH steps in 320 minutes, which is more than 5 hours.

Reply:
100 meetings in 5 rooms and 320 time grains (CH) takes ~20s on my machine. That's reasonable.
200 meetings in 5 rooms and 640 time grains (CH) takes ~200s on my machine. That's not reasonable but acceptable :) The larger sample data sets don't make sense to me.

We can discuss whether to include the large data sets later. This is just an observation, nothing blocking this PR.

Reply:
I think it's good to have both small examples to try fast and production-like data sets to test the difference of solution's quality between them (even if it takes too long, which mimics real world applications).
https://docs.optaplanner.org/7.7.0.Final/optaplanner-docs/html_single/index.html#doesPlannerFindTheOptimalSolution